build_support/jenkins.py

```diff
@@ -594,7 +594,10 @@ def write_summary(out_dir, completed_builds, ljen, failure=False):
         message = repo_set.repo(project).commit().message.splitlines()[0]
         # allow for special chars in message.
         git_log[project] = xml.sax.saxutils.quoteattr(message) 
-
+    rev_list = []
+    # rev_list is used for creating the retest build url in the summary
+    for (project, rev) in ljen._revspec._revisions.items():
+        rev_list.append("{}={}".format(project, rev))
```

Comment:
I swear I commented on this, and now my comment is gone....

If `ljen._revspec._revisions` is a dict we should use `.iteritems()` instead of `.items`

This could be simplified as:
```python
['{}={}'.format(p, r) for p, r in ljen._revspec._revisions.iteritems()`]
```